src/components/ui/Input.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+const Input = (props) => {
+    const {
+        handleChange,
+        value,
+        id,
+        name,
+        type,
+        isRequired = false,
+        placeholder,
+        label,
+        position = "top",
+        labelFontSize = "5xl",
+        labelFontWeight = "normal",
+        radius = "md",
+        inputWidthSize = "",
+        inputHightSize = "",
+        inputFontSize = "3xl",
+        inputFontWeight = "light",
+
+        ...rest
+    } = props;
+
+    const inputContainerClasses =
+        position === "left" || position === "right"
+            ? "flex items-center "
+            : "block ";
+
+    const inputClasses = `border border-solid border-gray-300 px-[20px] py-[20px] rounded-${radius} 
+    ${inputWidthSize ? `w-${inputWidthSize} ` : ""} ${
+        inputHightSize ? `h-${inputHightSize} ` : ""
+    } 
+  
+  text-${inputFontSize} font-${inputFontWeight}`;
+
+    const inputStyles = {
+        width: inputWidthSize,
+        height: inputHightSize,
+    };
+    return (
+        <div className={` ${inputContainerClasses}`}>
```

Comment:
It can be controlled with parent-container outside Input component, which will surround Input component, so remove it.